src/pages/registration/FilesAndLicensePanel.tsx

```diff
@@ -283,6 +346,16 @@ export const FilesAndLicensePanel = ({ uppy }: FilesAndLicensePanelProps) => {
           </Box>
         ))}
       </Modal>
+
+      <Modal
+        headingText={t('registration.files_and_license.embargo')}
+        open={isEmbargoModalOpen}
+        onClose={toggleEmbargoModal}
+        maxWidth="sm">
+        <Box>
+          <Typography>{t('registration.files_and_license.file_publish_date_helper_text')}</Typography>
+        </Box>
```

Comment:
Trenger sikkert ikke Box her?
```suggestion
        <Typography>{t('registration.files_and_license.file_publish_date_helper_text')}</Typography>
```